packages/layout/src/css/index.js

```diff
@@ -1,7 +1,3 @@
-import AAsideLayout from './aside-layout.js'
-import AEqualColumnLayout from './equal-column-layout.js'
-import APageHeadingLayout from './page-heading-layout.js'
-
-export const AsideLayout = AAsideLayout
-export const EqualColumnLayout = AEqualColumnLayout
-export const PageHeadingLayout = APageHeadingLayout
+export { default as asideLayoutCSS } from './aside-layout.js'
```

Comment:
Would it be simpler to just combine these into a single `css/index.js` file (and built .css and .scss file)?

Ie, `export  * from './aside-layout.js`, etc

Ehhh, probably not, right, because there aren't named exports in these files like `aside-layout.js`, nm...

Reply:
Sorta but I think we agreed to tackle this as separate PR and not break the current patterns. Want to pair on after your leadership offsite?